site/topics/overview/overview.rst

```diff
@@ -315,9 +315,88 @@ Examples
 
 
 
+Typical Settings
+================
+
+* Evolutionary computation algorithms have several hyperparameters to set
+* How many there are will depend on the specific type
+
+* For a generational GA using tournament selection that will run for some number of generations
+
+    * Number of generations
+
+        * As big as it needs to be
+        * Could be in the hundreds or the billions
+
+
+    * Population size
+
+        * Could be a few dozen or in the thousands
+
+
+    * Crossover rate (usually around 80%)
+
+        * Usually around 80%
+
+
+    * Mutation rate (usually around 10%)
+
+        * Usually around 10%
```

Comment:
usually around 10%  
usually around 10%

Reply:
![image](https://github.com/jameshughes89/cs4XX-EvolutionaryComputation/assets/1328048/8d91d33d-b138-411d-9a7e-9802c649dfd4)

Reply:
that was supposed to be a gif, but whoopsie...